File: Script/백준 13420.cpp
#include <iostream>

int main() {

	int t = 0;
	std::cin >> t;
	while (t > 0) {
		long long int a = 0, b = 0, result = 0;
		char operation, equal;
		std::cin >> a >> operation >> b >> equal >> result;

		if (operation == '+') {
			std::cout << (a + b == result ? "correct" : "wrong answer") << '\n';
		}
		else if (operation == '-') {
			std::cout << (a - b == result ? "correct" : "wrong answer") << '\n';
		}
		else if (operation == '*') {
			std::cout << (a * b == result ? "correct" : "wrong answer") << '\n';
		}
		else if (operation == '/') {
			std::cout << (a / b == result ? "correct" : "wrong answer") << '\n';
		}
		t--;
	}
}

File: 스언어_파이썬/백준_13420.py
testCase = int(input())
while testCase:
    line = list(map(str, input().split()))
    if line[1] == '*':
        if int(line[4]) == int(line[0]) * int(line[2]):
            print("correct")
        else:
            print("wrong answer")
    if line[1] == '/':
        if int(line[4]) == int(line[0]) / int(line[2]):
            print("correct")
        else:
            print("wrong answer")
    if line[1] == '+':
        if int(line[4]) == int(line[0]) + int(line[2]):
            print("correct")
        else:
            print("wrong answer")
    if line[1] == '-':
        if int(line[4]) == int(line[0]) - int(line[2]):
            print("correct")
        else:
            print("wrong answer")
    testCase -= 1
